### ISSUE TITLE
Config option for `ignore_history` from `bspc wm -h` flag

### ISSUE THREAD

**Issue by MyNameIsCosmo** [baskerville/bspwm#909]
I guess this is more of a style choice, but would it make sense to introduce a config option for `ignore_history`?
It seems like `bspc config ignore_history true` makes more sense than `bspc wm -h false`.

I could introduce a patch to have `ignore_history` handled by `bspc config`, and add a warning to deprecate `bspc wm -h false`, which would just point to the `bspc config` parameter.

Thoughts?

**Comment by baskerville** [comment]
Yes, you can make a pull request for this. Backward compatibility doesn't exist yet in this project: I'm using semantic versioning.
